models/place-bookingInfo.json

```diff
@@ -36,22 +38,30 @@
         "nl": {
           "type": "string",
           "description": "Dutch description",
-          "example": "Nederlandse beschrijving"
+          "example": "Nederlandse beschrijving",
+          "pattern": "\\S",
+          "minLength": 1
```

Comment:
Aside from these checks on `bookingInfo.urlLabel` that its values are not empty, we also need to check that `urlLabel` is set if `url` is set. See https://github.com/cultuurnet/udb3-silex/blob/6f629a3bbfa4f6caec085b8642d2a58ff05de522/src/Model/Validation/ValueObject/Contact/BookingInfoValidator.php#L27-L31 (it's a bit hard to read but that Respect Rule makes `urlLabel` required if `url` is set)

We can do that in this schema for `bookingInfo` using an `if/then` like we do for `calendarType: periodic` and `startDate`/`endDate`

Reply:
I checked and it's also not validated in [BookingInfoValidationRequestBodyParser.php](https://github.com/cultuurnet/udb3-silex/pull/905/files#diff-172d74d0f1cfc57407a12384eb9f9179f725794f4dc126c0a6510576e20ae8de) (but I'd prefer to do it in the schema anyway)

Reply:
There is also a property [dependentRequired](https://json-schema.org/understanding-json-schema/reference/conditionals.html#dependentrequired) and that seems to cover this case.

Reply:
`dependentRequired` is indeed better, good find!